components/SkillsCard.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+
+type CardProps = {
+  heading: string;
+  content: string;
+  imgSrcWhite?: string;
+  imgSrcBlue?: string;
+};
+
+const SkillsCard = ({
+  imgSrcWhite,
+  imgSrcBlue,
+  heading,
+  content,
+}: CardProps) => {
+  return (
+    <div className="group relative h-[16.5625rem] w-[18.125rem] rounded-md bg-white-900 p-4 transition duration-300 hover:translate-y-[2.8rem] hover:bg-primary-light dark:bg-black-200">
+      <div className="absolute left-[1.5625rem] top-[1.875rem] flex h-[3.875rem] w-[3.875rem] items-center justify-center rounded-lg bg-primary-light transition-colors duration-300 group-hover:bg-white">
+        <Image
+          src={imgSrcWhite || '/assets/redux.svg'}
+          alt={heading}
+          width="20"
+          height="20"
+          className="object-cover group-hover:hidden"
+        />
+        <Image
+          src={imgSrcBlue || '/assets/redux.svg'}
+          alt={heading}
+          width="20"
+          height="20"
+          className="hidden object-cover group-hover:block"
+        />
```

Comment:
If what you're doing here is rendering a different image (a black and white one and a colored one) on hover- have you considered just using the colored image and using the grayscale atomic class from tailwind?

https://tailwindcss.com/docs/grayscale

What you're doing is fine- but perhaps that works also for your use-case